### PR TITLE
Main patch tm-v0.34.11

### PR DESCRIPTION
**Pull request by tnasu**
#### STATUS: commit hash

- PICK: https://github.com/tendermint/tendermint/commit/1614e12035db528c7f43af2b28aea835ba7e505f
  - Fix codecov: https://github.com/line/ostracon/pull/359/commits/d13c9366e3041e51bda4acf8c7b387d70af11ced
- SKIP: https://github.com/tendermint/tendermint/commit/a9b4fac610c294e41361343cecefa95bd289db51
- PICK: https://github.com/tendermint/tendermint/commit/353e3a3243bcf0c49602259c415862a87b6a4f1a
  - Fix codecov: https://github.com/line/ostracon/pull/359/commits/48a55ae597b79b190a0b1761f0e70d3e0b944d46
- PICK: https://github.com/tendermint/tendermint/commit/8dd8a4e8ea2f3d214324166de10d211291cf888d
  - Fix lint: https://github.com/line/ostracon/pull/359/commits/dd361a269f84d7f93746a0f54b723fb0b63aa54b
- SKIP: https://github.com/tendermint/tendermint/commit/c376b44f1cefef04274a909df5c40c78d0a9b9a8
- PICK: https://github.com/tendermint/tendermint/commit/358b1f23c048dcea15cbf12e0053db7dc3e24ad8
- PICK: https://github.com/tendermint/tendermint/commit/be8c9833ca16c979bc3b0861bd9526e8503be336
- PICK: https://github.com/tendermint/tendermint/commit/d515bbcf1daf386b48247b76bb60f1adf889ecb2
- SKIP: https://github.com/tendermint/tendermint/commit/c2908ef785bb5ca6e015910368e4b7a32821b9ff

### CODE REVIEW

**Review by torao — APPROVED** [review]
LGTM

File: statesync/reactor.go [hunk 1]
```diff
@@ -59,8 +70,8 @@ func (r *Reactor) GetChannels() []*p2p.ChannelDescriptor {
 		},
 		{
 			ID:                  ChunkChannel,
-			Priority:            1,
-			SendQueueCapacity:   4,
+			Priority:            3,
```
[the remark this answers is not in the document]
Comment (tnasu) [reply statesync/reactor.go:73]: First, see the commit (relates 2 PRs): https://github.com/tendermint/tendermint/commit/d515bbcf1daf386b48247b76bb60f1adf889ecb2

Receiving chunks wait much time, so they improve chunk message priority.

[the remark this answers is not in the document]
Comment (kokeshiM0chi) [reply statesync/reactor.go:73]: Thank you!

**Review by kokeshiM0chi — APPROVED** [review]
LGTM

**Pull request merged**